pilot/pkg/security/authz/matcher/header_test.go

```diff
@@ -62,6 +63,88 @@ func TestHeaderMatcher(t *testing.T) {
 	}
 }
 
+func TestHostMatcher(t *testing.T) {
```

Comment:
I think we should use the new API in master fix, see the https://github.com/istio-private/istio/pull/71 for the fix, you can probably just cherry-pick it (it may have a little bit code conflict but should be easy to resolve).

Reply:
Cherry picked the old version